apps/ledger-live-mobile/.eslintrc.js

```diff
@@ -118,4 +118,20 @@ module.exports = {
     it: false,
     expect: false,
   },
+  overrides: [
+    {
+      files: ["e2e/*.js"],
+      globals: {
+        fetch: false,
+        it: false,
+        expect: false,
```

Comment:
Can we remove these now that we aren't exposing the detox globals? @valpinkman

Reply:
it's only for `e2e/*.js` files